### ISSUE TITLE
Create namespace support

### ISSUE THREAD

**Issue by christeoh** [cloudera/navigator-sdk#90]
Is there a way in the SDK to create namespaces?

**Comment by aadarshjajodia** [comment]
@christeoh yes i believe you can. Look at the CustomHiveLineageCreator.java class and see how it calls registerModels. I think that should be doing it. Let me know if you face issues in that.
